superdesk/factory/app.py

```diff
@@ -414,6 +416,24 @@ def setup_sentry(self):
 
         sentry_sdk.init(dsn=self.config["SENTRY_DSN"], integrations=[QuartIntegration(), AsyncioIntegration()])
 
+    def extend_eve_home_endpoint(self, links: list[dict]) -> None:
```

Comment:
This method is never called.

Reply:
You'll need to grab latest eve lib (from our superdesk/eve/Async repo & branch)